### PR TITLE
Actualizar documentación de telegestión para GIS

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

Update field firmware

### Tasks:

- [x] Add info firmware

### Links:

> Add links for each file modified

* [Distri/Gis/WebGis/Visor Web](http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/ca/gis/webgis/#capes-de-telegestio)
* [Distri/Gis/WebGis/Visor Web](http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/es/gis/webgis/#capas-de-telegestion)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: locales/es_ES/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -28876,6 +28876,9 @@ msgstr "Direccion del CUPS"
 msgid "Concentrador"
 msgstr "Concentrador"
 
+msgid "Versió de firmware / Versió de firmware PRIME"
+msgstr "Versión del firmware / Versiona del firmware PRIME"
```
Comment (jaumef) [inline locales/es_ES/LC_MESSAGES/messages.po]: Versiona → Versión

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_doc_gis_tg/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**